frontend/pages/occurrences/[slug].tsx

```diff
@@ -15,7 +15,7 @@ interface OccurrenceProps {
  */
 const Occurrence: FC<OccurrenceProps> = ({ occurrence }: OccurrenceProps) => {
   return (
-    <Layout title={""}>
+    <Layout title={occurrence["summary"] || occurrence["slug"]}>
```

Comment:
I think `occurrence["summary"]` is probably sufficient, since summary is a required field, but @mattfredericksen, if you think it's a good idea to use slug as a fallback, should we go ahead and use a method to replace the slug's hyphens with spaces?

Reply:
> Do you think we should truncate the summary here in case it's too long?

I think we should _not_ truncate it since it's a detail page (where else can someone see the full summary?), assuming the current design allows long summaries.

Reply:
@alashow Ah, I mean should we truncate the summary that is inserted into the page title, not the one that is displayed on the page. After a quick google search, it seems like we want our page titles to stay under 60 characters. I think we should either intelligently truncate the title (i.e. on words, not characters), enforce a summary length limit in the model, or add a new attribute for page titles.